tests/actions/PolicyTest.ts

```diff
@@ -162,9 +166,9 @@ describe('actions/Policy', () => {
             });
 
             // Check if the report pending action and fields were cleared
-            _.forEach(allReports, (report) => {
-                expect(report.pendingAction).toBeFalsy();
-                expect(report.pendingFields.addWorkspaceRoom).toBeFalsy();
+            Object.values(allReports ?? {}).forEach((report) => {
+                expect(report?.pendingAction).toBeFalsy();
+                expect(report?.pendingFields?.addWorkspaceRoom).toBeFalsy();
```

Comment:
Should we check here that report is not undefined? Otherwise these tests could be passing because `report?.pendingAction` and `report?.pendingFields?.addWorkspaceRoom` are undefined which is falsy

Reply:
I guess we can't remove the optional chaining operator since elements of allReports have the type OnyxCollection<Report>. Should we get rid of the check you just added above? Sorry for the back and forth here